packages/components/src/Avatar/_docs/Avatar.stickersheet.stories.tsx

```diff
@@ -17,6 +17,17 @@ export default {
   parameters: {
     chromatic: { disable: false },
     controls: { disable: true },
+    a11y: {
+      config: {
+        rules: [
+          {
+            // @todo: revisit this, stickersheet headings are causing color-contrast errors
+            id: "color-contrast",
+            enabled: false,
```

Comment:
This was a bit weird... @dougmacknz

Reply:
Test failures for reference: https://github.com/cultureamp/kaizen-design-system/actions/runs/6021336191/job/16334134544

Reply:
I think it just can't work out the background because of overlapping elements. Probs just ignore this as a false positive